packages/kolibri-tools/lib/cli.js

```diff
@@ -226,8 +227,15 @@ program
     list,
     []
   )
+  .option('--kds', 'Flag to use local kds', false)
   .option('--kds-path <kdsPath>', 'Full path to local kds directory', String, '')
   .action(function(mode, options) {
+    if (options.kds) {
+      if (!options.kdsPath) {
+        cliLogging.error('Path to the local KDS directory not specified.');
```

Comment:
I think updating this to indicate _why_ the error was received will help here. Like "The --require-kds-path flag was given but no --kds-path value was given. Please provide the local KDS directory" (or something like that).